Week7/app/src/main/java/com/mmoreno/pokeapp/networking/PokeService.kt

```diff
@@ -13,12 +13,15 @@ interface PokeService {
         const val BASE_URL = "https://pokeapi.co/api/v2/"
```

Comment:
Really nice work on this class and the model class above, nice idea getting only the first 100, this query could get very large otherwise.